resources/styles/components/course-calendar/month.less

```diff
@@ -122,9 +122,20 @@
 
       &--upcoming {
         cursor: pointer;
-
+        &:after{
+          content: '+';
+          font-size: 2rem;
+          position: absolute;
+          top: 0.5rem;
+          left: 0.5rem;
+          color: @tutor-tertiary-light;
+          opacity: 0;
```

Comment:
was gonna have them fade in, but i guess that might be a little much since the background already transitions.  ill put this in to the hover state